test/over_react/component/fixtures/context_type_component.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:over_react/over_react.dart';
+
+import 'test_context.dart';
+
+part 'context_type_component.over_react.g.dart';
+
+@Factory()
+UiFactory<ContextTypeProps> ContextType = _$ContextType;
+
+@Props()
+class _$ContextTypeProps extends UiProps {}
+
+@Component2()
+class ContextTypeComponent extends UiComponent2<ContextTypeProps> {
+
+  @override
+  final contextType = someContext.reactDartContext;
+
+  // --------------------------------------------------------------------------
+  // React Component Specifications and Lifecycle Methods
+  // --------------------------------------------------------------------------
+
+  @override
+  Map getDefaultProps() => (newProps());
+
+  @override
+  render() {
+    return Dom.span()('${this.context}');
+  }
+
+  // --------------------------------------------------------------------------
```

Comment:
Do we usually leave the empty sections? Or is it better to clean them up and remove them?

Reply:
Agreed... we should remove them.